lib/sisimai/bite/email/opensmtpd.rb

```diff
@@ -0,0 +1,192 @@
+module Sisimai::Bite::Email
+  module OpenSMTPD
+    # Sisimai::Bite::Email::OpenSMTPD parses a bounce email which created by
+    # OpenSMTPD. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/OpenSMTPD.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from     => %r/\AMailer Daemon [<][^ ]+[@]/,
+        :subject  => %r/\ADelivery status notification/,
+        :received => %r/[ ][(]OpenSMTPD[)][ ]with[ ]/,
+      }
+      # http://www.openbsd.org/cgi-bin/man.cgi?query=smtpd&sektion=8
+      # opensmtpd-5.4.2p1/smtpd/
+      #   bounce.c/317:#define NOTICE_INTRO \
+      #   bounce.c/318:    "    Hi!\n\n"    \
+      #   bounce.c/319:    "    This is the MAILER-DAEMON, please DO NOT REPLY to this e-mail.\n"
+      #   bounce.c/320:
+      #   bounce.c/321:const char *notice_error =
+      #   bounce.c/322:    "    An error has occurred while attempting to deliver a message for\n"
+      #   bounce.c/323:    "    the following list of recipients:\n\n";
+      #   bounce.c/324:
+      #   bounce.c/325:const char *notice_warning =
+      #   bounce.c/326:    "    A message is delayed for more than %s for the following\n"
+      #   bounce.c/327:    "    list of recipients:\n\n";
+      #   bounce.c/328:
+      #   bounce.c/329:const char *notice_warning2 =
+      #   bounce.c/330:    "    Please note that this is only a temporary failure report.\n"
+      #   bounce.c/331:    "    The message is kept in the queue for up to %s.\n"
+      #   bounce.c/332:    "    You DO NOT NEED to re-send the message to these recipients.\n\n";
+      #   bounce.c/333:
+      #   bounce.c/334:const char *notice_success =
+      #   bounce.c/335:    "    Your message was successfully delivered to these recipients.\n\n";
+      #   bounce.c/336:
+      #   bounce.c/337:const char *notice_relay =
+      #   bounce.c/338:    "    Your message was relayed to these recipients.\n\n";
+      #   bounce.c/339:
+      Re1 = {
+        :begin  => %r/\A[ \t]*This is the MAILER-DAEMON, please DO NOT REPLY to this e[-]?mail[.]\z/,
+        :rfc822 => %r/\A[ \t]*Below is a copy of the original message:\z/,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->